internal/commands/whoami/command.go

```diff
@@ -51,5 +53,24 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 	}
 
 	ui.Print(terminal.NewTextLog("Currently logged in user: %s (%s)", userDisplay, userSecret))
+
+	groups, groupsErr := clients.Atlas.Groups()
+	if groupsErr == nil {
+		headerProjectID := "Project ID"
+		headerProjectName := "Project Name"
+		tableRows := make([]map[string]interface{}, 0, len(groups))
+		for _, group := range groups {
+			tableRows = append(tableRows, map[string]interface{}{
+				headerProjectID:   group.ID,
+				headerProjectName: group.Name,
+			})
+		}
+		ui.Print(terminal.NewTableLog(
+			fmt.Sprintf("Projects available (%d)", len(groups)),
```

Comment:
is this potentially a **very large** list of projects?  wondering if it'd be desirable to make this output "opt-in" via a flag so that `realm-cli whoami` doesn't just start blasting users with more information than they're used to